Bots/DotNet/Skills/CodeFirst/TeamsWaterfallSkillBot/Dialogs/Cards/CardSampleHelper.cs

```diff
@@ -192,6 +192,21 @@ public static HeroCard CreateHeroCard()
             return heroCard;
         }
 
+        public static HeroCard CreateHeroCard(string type)
+        {
+            var heroCard = new HeroCard
+            {
+                Title = "BotFramework Hero Card",
+                Subtitle = "Microsoft Bot Framework",
+                Text = "Build and connect intelligent bots to interact with your users naturally wherever they are," +
+                       " from text/sms to Skype, Slack, Office 365 mail and other popular services.",
+                Images = new List<CardImage> { new CardImage("https://sec.ch9.ms/ch9/7ff5/e07cfef0-aa3b-40bb-9baa-7c9ef8ff7ff5/buildreactionbotframework_960.jpg") },
```

Comment:
Let's avoid external dependencies, move the image to the project so the tests are self contained.

Reply:
consolidated.